Validation/Geometry/test/dddvsdb/runDDDvsDBGeometryValidation.sh

```diff
@@ -31,14 +31,14 @@ cd ${workArea}
 set myDir=`pwd`
 echo $myDir
 
-cp $CMSSW_RELEASE_BASE/src/CondTools/Geometry/test/geometryxmlwriter.py .
+cp $CMSSW_RELEASE_BASE/src/CondTools/Geometry/test/writehelpers/geometryxmlwriter.py .
```

Comment:
in general `writehelpers` are intended to help writing payloads. The configurations in `test` are intended for tests.

Reply:
There are two versions of `geometryxmlwriter.py`. They differ only in the name of the output file. I chose the one that has a more meaningful output file name. There is a lot of work to be done to make these scripts more consistent and maintainable. The whole process of copying and editing scripts is very fragile and error-prone and should be reconsidered.

I invite follow-up PRs to this one.